library/manageiq.py

```diff
@@ -152,28 +178,31 @@ def update_provider(self, provider_id, provider_name, endpoints):
         """ Updates the existing provider with new parameters
         """
         try:
+            update_time = time.strftime(ManageIQ.pattern, time.gmtime())
```

Comment:
Is it possible to get `update_time` from the api response instead of getting it here?